examples/react-native/src/features/InAppMessaging/Demo/Demo.tsx

```diff
@@ -26,7 +26,7 @@ function DemoRadioGroup({ data, ...rest }: any) {
   return (
     <RadioGroup {...rest} labelStyle={[styles.radioGroupLabel, styles.label]}>
       {data.map((item: string) => (
-        <Radio key={item} label={item} size={20} value={item} />
+        <Radio key={item} label={item} size="small" value={item} />
```

Comment:
I changed the type for the `size` prop to match the [RWA](https://ui.docs.amplify.aws/react/components/radiogroupfield#sizes).